### DISCUSSION TITLE
<title>not receiving email authentication, not emails for password reset when i login with email and password

### DISCUSSION THREAD

**Discussion by jon1dev** [community/community#119407] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

-I suspect that my older accounts have been disabled, because i did not enable any MFA. 
-I can still enter username and password and the prompt displays that it sent email authorisation but i never get email so cannot login.
-This is the same for two long term accounts and i cant even get in to delete them and i have private repos id like to retrieve but cant.

**Comment by nearlythere** [comment]
Is that related to this? https://github.com/orgs/community/discussions/118995

**Comment by ebndev** [comment]
Hi @jon1dev,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

We appreciate your understanding!
